client/src/components/InputSelect/InputSelect.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import "./InputSelect.css";
+import PropTypes from "prop-types";
+import "../../App.css";
```

Comment:
You shouldn't need to do this anymore - Lorenzo made a PR that set this for storybook globally